packages/react-api/src/hooks/useTileFeatures.js

```diff
@@ -126,3 +131,14 @@ export default function useTileFeatures({
 
   return [onDataLoad, onViewportLoad, fetch];
 }
+
+// TODO: Comentario => https://github.com/visgl/loaders.gl/pull/2128
```

Comment:
I understand that this isn't a TODO, right? I mean, are you expecting this to change in the feature?
Btw, "comentario" ;)